src/core/internal/string.d

```diff
@@ -15,7 +15,17 @@ nothrow:
 
 alias UnsignedStringBuf = char[20];
 
-char[] unsignedToTempString(ulong value, return scope char[] buf, uint radix = 10) @safe
+/**
+Converts an unsigned integer value to a string of characters. This
+implemenation is a template so it can be used when compiling with -betterC
```

Comment:
Regarding DDOC:

- The first line is a "title", so it should usually be a one liner/sentence
- I **think** the preferred style is to have 1 blank line between sections (e.g. before `Params:` / `Returns:`)
- `implemenation`

Would `pragma(inline, true)` help in this case ?

Reply:
> Would pragma(inline, true) help in this case ?

Yeah, maybe.  The source code would be available in -betterC builds.  GDC doesn't currently implement `pragma(inline, true)`, though [I think it's coming](https://github.com/D-Programming-GDC/GDC/pull/587).  🤔 Yeah, let's go with that for now.  I can always change it to a template later if it doesn't work out.

Reply:
> The first line is a "title", so it should usually be a one liner/sentence

Yes everything up to the first blank line is considered the title and therefore should be short. After the blank line the description follows which can be long. 

> I think the preferred style is to have 1 blank line between sections (e.g. before Params: / Returns:)

I prefer that.